src/r5py/r5/transport_network.py

```diff
@@ -99,7 +99,7 @@ def from_directory(cls, path):
                     ),
                     RuntimeWarning,
                 )
-        except KeyError:
+        except IndexError:
```

Comment:
What prompted switching from `KeyError` to `IndexError`? Should we include both?

Reply:
It was wrong :D and the test showed that. Hooray for test-driven development